ui/src/components/RoleForm/index.jsx

```diff
@@ -192,14 +193,11 @@ export default class RoleForm extends Component {
             </Fragment>
           )}
           <ListItem>
-            <TextField
-              label="Description"
-              name="description"
+            <MarkdownTextArea
               onChange={this.handleInputChange}
```

Comment:
There seems to be a bug in the onChange handler.

STR:

1. Navigate to `https://deploy-preview-1447--taskcluster-web.netlify.com/auth/roles/github-admin%3A*` (deploy preview of your latest commit)
2. Click on the "Write" tab
3. Try to change the text

Expected: The text can be edited.
Actual: The text cannot be edited.